config/application.rb

```diff
@@ -18,6 +18,18 @@ class Application < Rails::Application
     # Custom directories with classes and modules you want to be autoloadable.
     config.autoload_paths += [Rails.root.join('lib')]
 
+    config.before_initialize do
+      # Load up Errbit::Config with values from the environment
+      require Rails.root.join('config/load')
+
+      config.secret_key_base = Errbit::Config.secret_key_base
+      config.serve_static_assets = Errbit::Config.serve_static_assets
+    end
+
+    initializer "errbit.mongoid", before: "mongoid.load-config" do
```

Comment:
Not sure. Do you think it should be after? I'll defer to your judgement on this since you seem to be well versed in mongoid.